templates/admin-header.php

```diff
@@ -5,42 +5,32 @@
  * @package Activitypub
  */
 
+global $settings_tabs;
```

Comment:
why do we have to make it global? and if so, shouldn't we prefix it, to avoid collisions?

Reply:
Fixed in [8b74a14](https://github.com/Automattic/wordpress-activitypub/pull/1256/commits/8b74a14babdf7d501a211fb44998efb5393671a5)